README.md

```diff
@@ -1,26 +1,81 @@
 # docker-timemachine
 A docker container to compile the lastest version of Netatalk in order to run a Time Machine server.
 
+
 ## Installation
 
+### Step 1 - Start the Server
+
 To download the docker container and execute it, simply run:
 
 ```
-$ docker run -h timemachine --name timemachine --restart=unless-stopped -d -v /route/to/your/timemachine:/timemachine -it -p 548:548 -p 636:636 odarriba/timemachine
+$ docker run -h timemachine --name timemachine --restart=unless-stopped -d -v /external_volume:/timemachine -it -p 548:548 -p 636:636 odarriba/timemachine
 ```
 
-To add users, just run this command:
+Replace `external_volume` with a local path where you want to store your data.
+
+As the image has been started using the `--restart=always` flag, it will start when the computers boots up.
+
+
+
+### Step 2 - Add a User
+
+To add a user, run:
 
 ```
 $ docker exec timemachine add-account USERNAME PASSWORD MOUNT_POINT [VOL_SIZE_MB]
 ```
 
 But take care that:
-* `MOUNT_POINT` should be an absolute path, preferably inside `/timemachine`, so it will be stored in your external volume.
+* `MOUNT_POINT` should be an absolute path, preferably set it to `"/timemachine"`, so it will be stored right in your external volume.
```

Comment:
Maybe here I would said something like `preferably set it inside "/timemachine" path, so it...` because like this it looks like the mount point should be exactly `/timemachine`